packages/utilities/gel-foundations/src/typography.js

```diff
@@ -1,11 +1,78 @@
-import { MEDIA_QUERY_TYPOGRAPHY } from './breakpoints';
+import { MEDIA_QUERY_TYPOGRAPHY } from '../breakpoints';
 
 const fontFamilyBase = ', Helvetica, Arial, sans-serif';
 
 export const GEL_FF_REITH_SERIF = `ReithSerif${fontFamilyBase}`;
 export const GEL_FF_REITH_SANS = `ReithSans${fontFamilyBase}`;
 export const GEL_FF_REITH_SANS_COND = `ReithSansCondensed${fontFamilyBase}`;
 
+function getTypeSizes(type = 'default', script = 'latin') {
```

Comment:
Consider setting the default type to an existing value in the script `.json` files.
I raise this because I haven't encountered a type, `default`, in the script `.json` files. As such, the function might end up setting `font-size` and `line-height` to `undefined`.

Reply:
Hey @alex-magana, take a look to the last commit. It should be fine now.